service-workers/service-worker/registration-updateviacache.https.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<title>Service Worker: Registration-updateViaCache</title>
+<script src="/resources/testharness.js"></script>
+<script src="resources/testharness-helpers.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/test-helpers.sub.js"></script>
+<script>
+  const updateViaCacheValues = [undefined, 'imports', 'all', 'none'];
+  const scriptUrl = 'resources/update-max-aged-worker.py';
+  const scope = 'resources/blank.html';
```

Comment:
To make clear they are global constants, could we name these globals like kUpdateViaCacheValues, kScope or else UPDATE_VIA_CACHE_VALUES, SCOPE, etc?